editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -4377,6 +4408,7 @@ Node3DEditorViewport::Node3DEditorViewport(Node3DEditor *p_spatial_editor, Edito
 	_edit.mode = TRANSFORM_NONE;
 	_edit.plane = TRANSFORM_VIEW;
 	_edit.snap = true;
+	_edit.show_rotation_line = true;
```

Comment:
This struct seems to be used only once, why does it have different init in the header and in the class constructor?

Reply:
Historic reasons, I guess. I did think about cleaning this up, but I thought it can wait after alpha2, just in case.